packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1135,6 +1283,10 @@ export class Combobox
     }
   }
 
+  private isAllSelected = (): boolean => {
```

Comment:
This, and similar variables, can be class functions instead of assigned bound functions since these are not event handlers and `this` is bound to the component instance at their call site. https://www.joshmorony.com/understanding-jsx-for-stencil-js-applications/#event-binding has some nice and concise info on this.